oci/jenkins.py

```diff
@@ -61,6 +61,9 @@ def system_tests(self, custom_repos=None, engine_version="master",
             suite_type (str): Suite type to run. For available suite types see
                 SUITE_TYPE popup menu at:
                 https://jenkins.ovirt.org/job/ovirt-system-tests_manual/build
+            coverage (str): Report tests coverage. For available coverage options
+                see COVERAGE popup menu at:
+                https://jenkins.ovirt.org/job/ovirt-system-tests_manual/build
```

Comment:
This should be bool argument, and we should convert the bool to "yes" and "no" in this layer.